tests/utils/test_run_dbt_cloud_job.py

```diff
@@ -2,7 +2,7 @@
 
 import pytest
 
-from utils.run_dbt_cloud_job import trigger_dbt_run, poll_dbt_run
+from utils.run_dbt_cloud_job import poll_dbt_run, trigger_dbt_run
```

Comment:
should this change be included in this PR?

Reply:
> Are the changes in files test* because of linter?

Exactly. I included some examples of changes that might be required if we're to adapt this. If the fixes suggested by the linters are not added, then builds will fail. I can think of two ways to go forward:
- Add linters and fixes in a single PR. This will solve the problem, but it's going to be a significant number of changes.
- Add linters only, without breaking the build on failure. This should be a temporary solution. Follow up PRs should introduce the changes piece by piece. Then we can force linters.

Reply:
oh, didnt see the Work in progress label.

Reply:
I'd say let's use the second approach. Linting is not urgent and/or breaking.

Reply:
Ok, going for running but not failing approach. We can gradually move towards the first approach by requiring linters to pass on files that have changes.